specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,861 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-11-01-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.App/containerAppsJobs": {
+      "get": {
+        "tags": [
+          "ContainerAppsJobs"
+        ],
+        "summary": "Get the Container Apps Jobs in a given subscription.",
+        "operationId": "ContainerAppsJobs_ListBySubscription",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
>v3 [](http://example.com/codeflow?start=69&length=2)

Recommend using current v4 common types. Comment applies throughout.

Reply:
@mentat9 we updated the references to /v4 by adding the types.json file to this PR, but can see that the Swagger validation fails. How we can address this?

Reply:
This may have been due to a bad change in v4 that has since been reverted. See if it goes away with a refresh from main (you can diff v3 and v4 and see that they are now the same). Either way is fine: I won't block on it.

Reply:
@mentat9 we created a different PR today to add the new/updated version to the "release-app-Microsoft.App-2022-11-01-preview" branch and got the same Breaking change validation error. We added the files in the "/specification/common-types/resource-management" from the main branch.
https://github.com/Azure/azure-rest-api-specs/pull/22114 

Also, wanted to know why the SDK validations were skipped for the given PR?